platform/admin/ui/pom.xml

```diff
@@ -51,7 +51,7 @@
         <dependency>
             <groupId>net.minidev</groupId>
             <artifactId>asm</artifactId>
-            <version>1.0.2</version>
+            <version>${asm.version}</version>
```

Comment:
Not sure this is correct. I believe the JSON library in use here wants this older version. I recall messing around with this asm stuff during the geotools upgrade.

Reply:
This isn't changing the value of the version; it's just using the existing property. Do you think this should be more explicit to avoid being upgraded when the property value changes?